test/anyOf_test.js

```diff
@@ -299,4 +299,48 @@ describe("anyOf", () => {
 
     expect(node.querySelector("select").value).eql("1");
   });
+
+  describe.only("Arrays", () => {
```

Comment:
@LucianBuzzo Can you change `describe.only` to `describe`?

Reply:
:+1: